### DISCUSSION TITLE
dfk-report.csv blank for my MM Address

### DISCUSSION THREAD

**Discussion by EdumonarThorsos** [pwillworth/dfkreport#10] [Q&A] [answered]
When I input my MetaMask address and let Lila do her thing, I try downloading the CSV and it yields blank results. I've even had friends try to generate it for me. Wondering if there is a transaction or something that is causing the issue.

**Comment by pwillworth** [comment, marked as answer]
Hi, sorry about that!  I saw the error getting kicked out in there, it was related to trying to export a csv for an old report that was generated previously and cached.  I put in a change that should resolve that if you could try again please.  If it still does not work for some reason, please try generating for a slightly different date range and export again and let me know if that works or not.

**Reply by EdumonarThorsos** [reply]
When I do the full year export, I now see something like this:
<img width="178" alt="Screen Shot 2022-03-28 at 4 25 09 PM" src="https://user-images.githubusercontent.com/102557153/160497559-71813752-c117-482c-a563-fc11abd53f62.png">

When I bumped the date to 2021-01-02, then that seemed to give me what I needed.

**Comment by hysticali** [comment]
I seem to be having this same issue with DFK Chain when selecting the Koinly or Coinledger report. I am able to download the Manual CSV no problem and I had also downloaded the Harmony for Koinly.

**Reply by pwillworth** [reply]
Hi @hysticali thanks for the heads up.  Looks like a bug was introduced with recent updates for the new Jeweler events that was impacting CSV export.  It is fixed now, so you should be able to try again with success.
